webapp/transport/mixins.py

```diff
@@ -0,0 +1,12 @@
+from braces.views._access import AccessMixin
+from django.core.exceptions import PermissionDenied
+
+
+class UserIsDrivingMixin(AccessMixin):
```

Comment:
Why did you place this logic in a Mixin, were you planning on reusing it?

Reply:
Not really, I saw the pattern somewhere else and it seemed like a nice system to follow. I don't intent to reuse it, since it depends on specific fields. Would you advice including it in the view directly?

Reply:
Mixins are meant for reusing logic, so it doesn't necessarily make sense to use them like this. But it's fine like this as far as I'm concerned.